src/frontend/src/optimizer/plan_node/stream_eowc_over_window.rs

```diff
@@ -58,6 +58,7 @@ impl StreamEowcOverWindow {
             true,
             true,
             watermark_columns,
+            input.columns_monotonicity().clone(),
```

Comment:
Is this wrong? like the comment on `EowcOverWindowExecutor` cannot produce any watermark columns

Reply:
Well, EOWC has `ORDER BY`, so I assume if a non-order-by column is monotonic, it will break.

Reply:
Oh, seems we cannot derive monotonicity for any column in this case, just like for watermark, basically for the same reason.